bin/tag-release-continue.js

```diff
@@ -24,11 +25,21 @@ const callback = async options => {
 	if (options.branch.includes("promote-release")) {
 		flow = filterFlowBasedOnDevelopBranch(options, promoteContinue);
 
+		options.scripts = await utils.getScripts("promote");
```

Comment:
The `await` keyword can be removed as this is not an `async` function (both here and on line `38`).